pgactivity/data.py

```diff
@@ -249,7 +249,7 @@ def pg_terminate_backend(self, pid: int) -> bool:
         cur = self.pg_conn.cursor()
         cur.execute(query, {"pid": pid})
         ret: Dict[str, bool] = cur.fetchone()
-        return ret["terminated"]
+        return ret[0]
```

Comment:
Depending on the version it could be `is_terminated` or `is_cancelled`.
I'll settle it with another name (eg : `is_stopped` for the two queries).

Why doesn't pytest breaks in my computer ?

Reply:
> Why doesn't pytest breaks in my computer ?

I suspect this is because the psycopg2 connection is configured to use a 
DictCursor, so `ret[0]` is not valid since `ret` is a dict.

Using the same name in both queries should work, I believe.

Reply:
done